inductiva/storage/README.md

```diff
@@ -0,0 +1,46 @@
+# Storage Management
+
+The Inductiva API provides you with the tools to manage your cloud storage effectively. Currently, we support Google Cloud, but we're working on expanding our services to include multiple cloud platforms soon!
+With the Inductiva storage module, you can easily navigate your storage bucket, evaluate the space used, and delete specific directories as needed.
+Your storage bucket is organized by task ID, with each task corresponding to a folder named after its ID. These folders contain both the input and output files for each simulation.
```

Comment:
same

Reply:
Does it make sense to provide this context if we are going to change it soon? Shouldn't you mention already the new description?